docs/modules/ROOT/pages/spring-cloud-commons/application-context-services.adoc

```diff
@@ -41,12 +41,12 @@ If you build an application context from `SpringApplication` or `SpringApplicati
 It is a feature of Spring that child contexts inherit property sources and profiles from their parent, so the "`main`" application context contains additional property sources, compared to building the same context without Spring Cloud Config.
 The additional property sources are:
 
-* "`bootstrap`": If any `PropertySourceLocators` are found in the bootstrap context and if they have non-empty properties, an optional `CompositePropertySource` appears with high priority.
+* "`bootstrap`": If any `PropertySourceLocator` are found in the bootstrap context and if they have non-empty properties, an optional `CompositePropertySource` appears with high priority.
```

Comment:
This still needs some kind of plural. Instances?

Reply:
🤷🏻‍♂️. It still needs to be plural.

Reply:
OK